api/v1/misc.js

```diff
@@ -32,6 +32,31 @@ router.get('/status', async (req, res) => {
   }
 });
 
+router.get('/active', async (req, res) => {
+  const cache = await req.app.locals.cache.fetch(
+    {
+      key: req.originalUrl,
+      interval: 86400000, // 24 hours
+    },
+    async () => {
+      try {
+        const activeDays = await req.app.locals.query(
+          'SELECT COUNT(DISTINCT charid) AS active_players, CAST(login_time AS DATE) AS date FROM audit_iprecord WHERE login_time BETWEEN DATE_SUB(CURRENT_TIMESTAMP, INTERVAL 14 DAY) AND CURRENT_TIMESTAMP GROUP BY date;'
+        );
+        if (activeDays == null || activeDays.length < 1) {
+          return res.send(404);
+        }
+        const sum = activeDays.reduce((s, day) => s + parseInt(day.active_players, 10), 0);
+        res.send(Math.round(sum / activeDays.length).toString());
+      } catch {
+        return res.send(500);
```

Comment:
At least the two error status codes need to be sent with `res.status(<error-code>).send()` similar to other places, or else it will be considered an OK response with a body being the numbers. And the OK response could use a 200 status code defined as well for good measure.

Reply:
Whoops! Thanks for the careful review.

Reply:
Updated. Here is the fail state.
<img width="923" alt="fail" src="https://user-images.githubusercontent.com/6099499/189642319-ffabbe9f-b32a-4a65-8e6c-eb7e1fc4bbe3.png">